src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Oct 30, 2023
+tags: CI, CD, github, githubAction
+banner: https://blog.jankaritech.com/src/assets/githubAction/images/githubaction_banner.png
+---
+GitHub Actions is an automation and CI/CD service provided by GitHub. It allows us to automate our software development process or workflows, from building and testing code to deploying it to different environments. It's a really effective tool that can both increase software quality and save us time.
+
+## Some features of GitHub Actions:
+- It allows you to automate your Software Development Life Cycle (SDLC) processes or workflows.
+- It simplifies CI/CD integration, enabling seamless DevOps practices.
+
+## Components of GitHub Actions:
+1. Workflows
+2. Events
+3. Jobs
+4. Steps
+5. Actions
+6. Runners
+
+![components](/src/assets/githubAction/images/components_of_github_action.png)
+
+
+### 1. Workflows:
+Workflows are automated procedures that you add to your repository. Workflows will run when triggered by an event in your repository, or they can be triggered manually, or at a defined schedule. It can be used to build, test, package, release, or deploy a project on GitHub.
+
+### 2. Events:
+Events are specific activities like code pushes, pull requests creation, issue creation, issue comments, etc. that triggers a workflow.
```

Comment:
```suggestion
Events are specific activities like code push, pull requests creation, issue creation, comments on pull requests or issues, etc. that triggers a workflow.
```
if you mean like this

Reply:
maybe put "pull request creation" also into singular (instead of pull request**s** creation) since the other items are also singular